actions/websocket_actions.jsx

```diff
@@ -149,8 +151,18 @@ export function reconnect(includeWebSocket = true) {
 
     const currentTeamId = getState().entities.teams.currentTeamId;
     if (currentTeamId) {
+        const state = getState();
+        const currentChannelId = getCurrentChannelId(state);
+        const mostRecentId = getMostRecentPostIdInChannel(state, currentChannelId);
+        const mostRecentPost = getPost(state, mostRecentId);
         dispatch(loadChannelsForCurrentUser());
-        dispatch(getPosts(getCurrentChannelId(getState())));
+        if (mostRecentPost) {
+            dispatch(getPostsSince(currentChannelId, mostRecentPost.create_at));
+        } else {
+            // if network timesout the first time when loading a channel
```

Comment:
nit: timeout
```suggestion
            // if network timeout the first time when loading a channel
```

Reply:
I thought timesout is a word. Sounds like it should be `timed-out`